freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -20,13 +20,108 @@
 # SOFTWARE.
 #
 # Written by Valerij Maljulin <vmaljuli@redhat.com>
+# Written by Chuang Zhang <chuazhan@redhat.com>
 
-from freshmaker.handlers import ContainerBuildHandler
+import requests
+from http import HTTPStatus
+
+from freshmaker import conf, db, log
+from freshmaker.errata import Errata
 from freshmaker.events import FlatpakModuleAdvisoryReadyEvent
+from freshmaker.handlers import ContainerBuildHandler, fail_event_on_handler_exception
+from freshmaker.kojiservice import koji_service
+from freshmaker.lightblue import LightBlue
+from freshmaker.models import Event
+from freshmaker.pyxis import Pyxis
+from freshmaker.types import EventState
 
 
 class RebuildFlatpakApplicationOnModuleReady(ContainerBuildHandler):
     name = 'RebuildFlatpakApplicationOnModuleReady'
 
     def can_handle(self, event):
         return isinstance(event, FlatpakModuleAdvisoryReadyEvent)
+
+    @fail_event_on_handler_exception
+    def handle(self, event):
+        """
+        TODO: Update in future.
```

Comment:
Leftover TODO.

Reply:
Do you mean to remove the docstring totally? And we can add it when everything is ready?

Reply:
I am thinking to leave a placeholder there and update it in future.

Reply:
Not sure if there is anything worth noting - if not, I'm OK with removing it.

BTW, since this is on code review and not marked as draft, I assume it's ready. :)

Reply:
Yeah. It is ready. The thing is this handle method will be added more implementations for future tickets. So I am assuming to place a holder here to remind me to change it in future. Maybe it's worthless, let's remove it for no confusion :D